scripts/Build.ps1

```diff
@@ -170,6 +172,21 @@ function Perform-Restore {
   if ($lastExitCode -ne 0) {
     throw "The restore failed with an exit code of '$lastExitCode'."
   }
+  
+  $msbuild = Join-Path $msbuildPath "MSBuild.exe"
+  
+  Write-Verbose "Starting restore for NetCore Projects"
+  foreach($project in $TFB_NetCoreProjects)
+  {
+	$projectPath = Locate-Item -relativePath $project
+
+	Write-Verbose "$msbuild /t:restore -verbosity:quiet $projectPath"
+	& $msbuild /t:restore -verbosity:quiet $projectPath
```

Comment:
>quiet [](start = 34, length = 5)

can we make this minimal so we know whats going on instead of it looking like its stuck. Might want to change it on top too.